src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -55,11 +58,14 @@ public class UnsolvedSymbolVisitor extends ModifierVisitor<Void> {
    * The unsolved superclass of this current class file. If there is no unsolved superclass, then
    * the value of this variable is an empty string
    */
-  private @ClassGetSimpleName String superClass = "";
+  private final Map<String, @ClassGetSimpleName String> classAndItsParent = new HashMap<>();
 
   /** The package of this class */
   private String currentPackage = "";
 
+  /** The name of the class currently visited */
+  private String className = "";
```

Comment:
Is this a simple or fully-qualified name? (The docs should say, or you could document it with an annotation.)